src/interpreter/mod.rs

```diff
@@ -14,14 +14,15 @@
 //!
 //! The pact bytecode interpreter
 //!
+pub mod type_cast;
```

Comment:
Can we move this `mod` under `types` and export the `AnyTryInto` trait so that it is available through `interpreter::types::AnyTryInto`, now that we aren't gating `type_cast`

Reply:
@holygits Adding this line `pub use crate::interpreter::type_cast::AnyTryInto;` in `src/interpreter/types.rs` results in unused import warning 😞

Reply:
Can add an `#[allow(unused_import)]` to silence the warning

Reply:
The warning is gone now